Specifications/Language/3_Expressions/FunctorApplication.md

```diff
@@ -1,22 +1,23 @@
-Both have the same type and upon invocation execute the implementation defined for the `controlled adjoint` [specialization](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/4_SpecializationDeclarations.md#specialization-declarations).
+Both have the same type and, upon invocation, execute the implementation defined for the `controlled adjoint` [specialization](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/4_SpecializationDeclarations.md#specialization-declarations).
 
-## Adjoint Functor
+## Adjoint functor
 
-If the operation `ApplyUnitary` defines a unitary transformation *U* of the quantum state, `Adjoint ApplyUnitary` accesses the implementation of *U†*. The `Adjoint` functor is its own inverse, since *(U†)† = U* by definition; i.e. `Adjoint Adjoint ApplyUnitary` is the same as `ApplyUnitary`.
+If the operation `ApplyUnitary` defines a unitary transformation *U* of the quantum state, `Adjoint ApplyUnitary` accesses the implementation of *U†*. The `Adjoint` functor is its own inverse, since *(U†)† = U* by definition. For example, `Adjoint Adjoint ApplyUnitary` is the same as `ApplyUnitary`.
+
+The expression `Adjoint ApplyUnitary` is an operation of the same type as `ApplyUnitary`; it has the same argument and return type and supports the same functors. Like any operation, it can be invoked with an argument of suitable type. The following expression applies the [adjoint specialization](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/4_SpecializationDeclarations.md#specialization-declarations) of `ApplyUnitary` to an argument `arg`:
 
-The expression `Adjoint ApplyUnitary` is an operation of the same type as `ApplyUnitary`; it has the same argument and return type and supports the same functors. Like any operation, it can be invoked with an argument of suitable type. The following expression will apply the [adjoint specialization](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/4_SpecializationDeclarations.md#specialization-declarations) of `ApplyUnitary` to an argument `arg`:
 ```qsharp
 Adjoint ApplyUnitary(arg) 
 ```
 
-## Controlled Functor
+## Controlled functor
 
-For an operation `ApplyUnitary` that defines a unitary transformation *U* of the quantum state, `Controlled ApplyUnitary` accesses the implementation that applies *U* conditional on all qubits in an array of control qubits being in the |1⟩ state. 
+For an operation `ApplyUnitary` that defines a unitary transformation *U* of the quantum state, `Controlled ApplyUnitary` accesses the implementation that applies the *U* conditional on all qubits in an array of control qubits being in the |1⟩ state. 
```

Comment:
Typically one doesn't use "the" before the name of a unitary; either I'd remove it, or change to "the *U* transformation".

Reply:
```suggestion
For an operation `ApplyUnitary` that defines a unitary transformation *U* of the quantum state, `Controlled ApplyUnitary` accesses the implementation that applies *U* conditional on all qubits in an array of control qubits being in the |1⟩ state. 
```